sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Config/BatchOptions.cs

```diff
@@ -1,27 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-using System;
-
-namespace Microsoft.Azure.WebJobs.ServiceBus
-{
-    /// <summary>
-    /// Configuration options for ServiceBus batch receive.
-    /// </summary>
-    public class BatchOptions
```

Comment:
Why drop this type?

Reply:
The OperationTimeout is covered by MaxWaitTime which also applies to single message functions. Same with AutoComplete.
The only property that is specific to batching is the MaxMessageCount.